README.md

```diff
@@ -22,9 +22,21 @@ In the specifications in the above table of contents, the keywords "MUST", "MUST
 
 To provide context for users the following section gives example use cases for each part of the spec.
 
-## Filesystem Bundle & Configuration
+#### Application Bundle Builders
 
-- A user can create a root filesystem and configuration, with low-level OS and host specific details, and launch it as a container under an Open Container runtime.
+Application bundle builders can create a [bundle](bundle.md) directory that includes all of the files and some low-level details for launching an application as a container.
+The application can in-turn rely on certain OS configuration being available when it is launched inside of a container by a runtime.
```

Comment:
On Thu, Dec 24, 2015 at 06:24:56AM -0800, Jonathan Boulle wrote:
> >  
> > -- A user can create a root filesystem and configuration, with low-level OS and host specific details, and launch it as a container under an Open Container runtime.
> > +Application bundle builders can create a [bundle](bundle.md) directory that includes all of the files and some low-level details for launching an application as a container.
> > +The application can in-turn rely on certain OS configuration being available when it is launched inside of a container by a runtime.
> 
> It's pretty unclear what low level configuration the user (builder)
> must specify and what will be provided by something else (presumably
> the runtime).

I'm pretty sure this is calling attention to the requirements added in
in #164.  And again, I don't think we need to get down to this level
of detail here.  Folks are packaging some software in a container
because they want it to be easy to run in a reliable, safe manner.
They pick the OCI format because it's a popular (hopefully ;) standard
with broad (hopefully ;) support.  Making a low-level sales pitch for
the OCI format (vs. other formats?) seems to be off-target for a
doc-section where we're trying to paint pictures of who these
spec-user classes are.